router/functionHandler.go

```diff
@@ -93,6 +95,12 @@ func (fh *functionHandler) tapService(serviceUrl *url.URL) {
 func (fh *functionHandler) handler(responseWriter http.ResponseWriter, request *http.Request) {
 	reqStartTime := time.Now()
 
+	// retrieve url params and add them to request header
+	vars := mux.Vars(request)
+	for k, v := range vars {
+		request.Header.Add(fmt.Sprintf("X-Fission-Params-%v", k), v)
```

Comment:
Thanks for checking. I finally realized that the ": " would show up in the value, not the key, and the key itself can never have a ": ".